builder/vmware/common/step_shutdown.go

```diff
@@ -13,30 +13,76 @@ import (
 	"strings"
 	"time"
 
+	"github.com/hashicorp/packer-plugin-sdk/communicator"
 	"github.com/hashicorp/packer-plugin-sdk/multistep"
+	"github.com/hashicorp/packer-plugin-sdk/packer"
 	packersdk "github.com/hashicorp/packer-plugin-sdk/packer"
 )
 
-// StepShutdown shuts down the machine. It first attempts to do so gracefully,
-// but ultimately forcefully shuts it down if that fails.
+//go:generate packer-sdc struct-markdown
+
+const (
+	defaultShutdownTimeout = 5 * time.Minute
+	pollInterval           = 150 * time.Millisecond
+	postShutdownTimeout    = 120 * time.Second
+	fileReleaseDelay       = 5 * time.Second
+)
+
 type StepShutdown struct {
 	Command string
 	Timeout time.Duration
-
-	// Set this to true if we're testing
 	Testing bool
+	Disable bool
+}
+
+type ShutdownDisableConfig struct {
+	// Disables the default shutdown process. This is useful for debugging.
```

Comment:
Given the rationale for the option, I'm not sure this is advised.
There's already multiple ways to handle errors and allow debugging a failed build, notably command-line flags like `--debug` which waits for user input after every step, or `--on-error` which changes how Packer behaves in case of error, allowing some command like `packer build --on-error wait`, which waits for user-input when something fails during the build.
Is there another reason for this addition?

Reply:
Closing pull request.